lms/djangoapps/courseware/access.py

```diff
@@ -292,36 +297,27 @@ def can_load():
 
         NOTE: this is not checking whether user is actually enrolled in the course.
         """
-        # delegate to generic descriptor check to check start dates
-        return _has_access_descriptor(user, 'load', course, course.id)
+        response = (
+            _visible_to_nonstaff_users(courselike) and
+            _can_access_descriptor_with_start_date(user, courselike, courselike.id)
+        )
+
+        return (
+            ACCESS_GRANTED if (response or _has_staff_access_to_descriptor(user, courselike, courselike.id))
+            else response
```

Comment:
Is "nonstaff_access_response" a more clear name for "response"?  Also, I think it may be more clear to drop response from the first if:
```
   ACCESS_GRANTED if _has_staff_access_to_descriptor(user, courselike, courselike.id)
   else nonstaff_access_response
```